### PR TITLE
Make it work with array as columns

### PR DESCRIPTION
**Pull request by jeremyvergnas**
- Make columns usable as array of objects
- Make populate usable as array of strings

### CODE REVIEW

File: doc/usage.md [hunk 1]
```diff
@@ -22,7 +22,25 @@ this.repository = entityManager.getRepository('users');
...
 
 ### columns
-Comma-separated string representing the column names to display. This is used for table content, but also the table headers. There's support for nested objects, as well as aliases. Example:
+Comma-separated string representing the column names to display. Or Array of objects describing the columns. Example:
+```html
+<datatable resource="user" columns.bind="columns"></datatable>
+```
+```js
+export class List {
+  columns = [{
```
Comment (RWOverdijk) [inline doc/usage.md:31]: Can't I just do `'id'`?

File: src/datatable.js [hunk 2]
```diff
@@ -87,6 +87,9 @@ export class DataTable {
     } else if (typeof this.populate === 'string') {
       this.criteria.populate = this.populate;
     }
+    else if(Array.isArray(this.populate)){
```
Comment (RWOverdijk) [inline src/datatable.js]: Else goes on same line as closing bracket

File: src/datatable.js [hunk 3]
```diff
@@ -232,6 +231,23 @@ export class DataTable {
       return str[0].toUpperCase() + str.substr(1);
     }
 
+    if (Array.isArray(this.columns)) {
+      let labels = this.columns.map(column => {
```
Comment (RWOverdijk) [inline src/datatable.js]: Just return the map?

**Review by RWOverdijk — CHANGES_REQUESTED** [review]

File: doc/usage.md (hunk 1 above)
Comment (jeremyvergnas) [reply doc/usage.md:31]: Wanted to show that we can used nested object. Maybe `group.name` is a better example then

Comment (RWOverdijk) [reply doc/usage.md:31]: 👍

**Comment by jeremyvergnas** [comment]
@RWOverdijk Done

**Review by RWOverdijk — APPROVED** [review]

**Comment by RWOverdijk** [comment]
@doktordirk Could you double check? :)

**Comment by doktordirk** [comment]
@RWOverdijk looks fine

**Review by bas080 — APPROVED** [review]

**Review by VMBindraban — APPROVED** [review]

**Comment by VMBindraban** [comment]
Tested and works fine!

**Pull request merged**